[16:23:01] [main/INFO]: Python script exited with code: 2
